git-version-macro/src/lib.rs

```diff
@@ -101,17 +101,47 @@ pub fn git_describe(input: TokenStream) -> TokenStream {
 }
 
 #[proc_macro_hack]
-pub fn git_version(input: TokenStream) -> TokenStream {
-	parse_macro_input!(input as Nothing);
+pub fn git_describe_or_cargo_version(input: TokenStream) -> TokenStream {
+	let args: Vec<_> = parse_macro_input!(input as ArgList).args.iter().map(|x| x.value()).collect();
 
-	let tokens = match git_describe_impl(&VERSION_ARGS) {
+  let cargo_version = concat!(env!("CARGO_PKG_VERSION"), "-cargo");
```

Comment:
This would give the Cargo version of the git-version-macro crate itself :P

Reply:
You're right, it's fixed now. Funny how easy it is to make that kind of mistake.  I tested it, but was only looking to see if the "-cargo" showed up, and must have ignored the version number.  Sadly I don't know an easy way to introduce an integration test for this, since any such test would live in a git repository normally.

Reply:
It is possible to break git using a build script by setting the `GIT_DIR` environment variable to something that isn't a git directory. That in a dedicated crate would probably work.